tools/disasm/variables.txt

```diff
@@ -2269,6 +2269,8 @@
     0x801D703C:("sOcarinaMemoryGameEndPos","UNK_TYPE1","",0x1),
     0x801D7040:("sOcarinaMemoryGameNumNotes","UNK_TYPE1","",0x1),
     0x801D7044:("sOcarinaSongNotes","OcarinaNote","[24][20]",0xF00),
+    0x801D7E04:("D_801D7E04","UNK_TYPE1","",0x1),
+    0x801D7EA4:("D_801D7EA4","UNK_TYPE1","",0x1),
```

Comment:
Can you revert this file back? Looks like it got added by accidently from a previously closed PR.

Reply:
reverted